packages/components/src/mobile/bottom-sheet/index.native.js

```diff
@@ -31,14 +34,62 @@ import CyclePickerCell from './cycle-picker-cell';
 import PickerCell from './picker-cell';
 import SwitchCell from './switch-cell';
 import RangeCell from './range-cell';
+import ColorCell from './color-cell';
 import KeyboardAvoidingView from './keyboard-avoiding-view';
 
+const ANIMATION_DURATION = 300;
+
+// Context in BottomSheet is necessary for controlling the
+// transition flow between subsheets and replacing a content inside them
+export const { Provider: BottomSheetProvider, Consumer } = createContext( {
```

Comment:
You export the `Consumer` two times
https://github.com/WordPress/gutenberg/pull/21326/files#diff-f2a9744b0d8b570033a66a2f59c56610R459

```suggestion
export const { Provider: BottomSheetProvider, Consumer: BottomSheetConsumer } = createContext( {
```

Reply:
Done